vue-components/src/components/HelloWorld.vue

```diff
@@ -38,17 +38,30 @@ export default Vue.extend( {
 
 <style lang="scss">
 .wikit-HelloWorld {
+	inset: logical 0 5px 10px;
+
 	&__headline {
-		margin: 40px 0 0;
+		margin-block-start: 40px;
+		margin-inline-end: 0;
+		margin-block-end: 0;
 	}
+
 	&__action-list {
 		list-style-type: none;
-		padding: 0;
+		padding-block-start: 0;
+		padding-inline: 0;
+		padding-block-end: 0;
+		padding-inline-end: 0;
 	}
+
 	&__action {
 		display: inline-block;
-		margin: 0 10px;
+		margin-block: 0;
+		margin-inline-end: 10px;
+		margin-block-end: 0;
+		margin-inline: 10px;
```

Comment:
From my interpretation a sufficient conversion would have been
```suggestion
		margin-block: 0;
		margin-inline: 10px;
```
(margin-inline does "left" and "right" at once, no need for the margin-inline-end). Do you share this view or am I still misunderstanding something about this new field?

And with #117 it will only be (**Not really relevant - just for alignment**)

```suggestion
		margin-inline: 10px;
```